mohawk/tests.py

```diff
@@ -798,15 +811,6 @@ def test_validate_bewit_with_ext(self):
         })
         self.assertTrue(check_bewit(url, credential_lookup=credential_lookup, now=1356420407 + 10))
 
-    def test_validate_bewit_with_ext_and_backslashes(self):
```

Comment:
And the same goes for this test: can the assertions be changed to expect an error?